ehitchhiking-ui/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
you forgot about useTabs property (https://prettier.io/docs/en/options.html#tabs). I recommend you to set true value

Reply:
the same for arrow function https://prettier.io/docs/en/options.html#arrow-function-parentheses.  I recommend you to set "always" value

Reply:
done